site/en/docs/webstore/publish/index.md

```diff
@@ -33,25 +33,23 @@ We'll go into detail about each step below.
 
 ## Create your item's zip file {: #create-your-items-zip-file }
 
-To upload your item, you need to create a ZIP file that contains the files for your extension. The
-item's manifest file must be included, and it must specify at least the following fields:
+To upload your item, you need to create a ZIP file that contains the 
+manifest file located in the **root** and the files for your extension. The manifest file must specify at least the following fields:
```

Comment:
root --> root directory

(Well, relative to the package I guess. But not just "in the root".)